packages/typespec-client-generator-core/lib/decorators.tsp

```diff
@@ -446,6 +446,11 @@ enum InitializedBy {
    * The client could be initialized by parent client.
    */
   parent: 2,
+
+  /**
+   * The client can't be initialized.
+   */
+  none: 4,
```

Comment:
I just think of a question, can we mark the initializedBy as both none and (parent or individually)? I feel like none and other values are mutual exclusive?

Reply:
Yeah, none could only used independently and here is logic to check: https://github.com/Azure/typespec-azure/blob/90b771eef9f262150e111c80e9c1f5e1e900aaa0/packages/typespec-client-generator-core/src/clients.ts#L323-L333

Reply:
I wonder if we can put None as 0, because if it's 0, then 
```
None | Parent = Parent
None | Individually = Individually
```
in this way we don't need to do extra checks maybe?

Reply:
Wondering whether `default` or `not_specified` be a better name.

Reply:
> I wonder if we can put None as 0, because if it's 0, then
> 
> ```
> None | Parent = Parent
> None | Individually = Individually
> ```
> 
> in this way we don't need to do extra checks maybe?

I am afraid the logic can't apply for all statements: https://github.com/Azure/typespec-azure/blob/90b771eef9f262150e111c80e9c1f5e1e900aaa0/packages/typespec-client-generator-core/src/clients.ts#L294-L337

Reply:
> Wondering whether `default` or `not_specified` be a better name that describes this config.

Yeah, `default` may be more general and not that misleading.